plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -1637,6 +1654,13 @@ def validate_radio_parameters(self, radio_config, radio_series, errormsg):
         if antenna_gain and antenna_gain not in range(0, 41):
             errormsg.append("antenna_gain: Invalid '{0}' in playbook".format(antenna_gain))
 
+        cable_loss = radio_config.get("cable_loss")
+        if cable_loss and cable_loss not in range(0, 41):
```

Comment:
cable_loss is common in both if and elif, hence you can optimize it

Also you can specify the value in errormsg..

```
if cable_loss:
    if not 0 <= cable_loss <= 40:
        errormsg.append("cable_loss: Invalid '{0}' in playbook. Must be between 0 and 40.".format(cable_loss))
    elif antenna_gain and cable_loss >= antenna_gain:
        errormsg.append("cable_loss: Invalid '{0}' in playbook. Must be less than antenna_gain: {1}.".format(cable_loss, antenna_gain))
```

Reply:
Updated as per above